src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let money = days * 40;
+
+  if (days >= 7) {
+    money = money - 50;
+  }
```

Comment:
After the 'if' statement with a return, you don't need to use 'else' for the following conditions. The code can be simplified by removing the 'else' and having a single return statement at the end.